components/ProductsList.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="category-area">
+    <NuxtLink
+      to="/"
+      v-for="category in categories"
+      :key="category.id"
+      class="category-link"
+      :style="{ backgroundImage: `url(${category.bgUrl})` }"
+    >
+      <div class="info-area">
+        <p class="product-name">{{ category.name }}</p>
+
+        <div>
+          <p class="price price-old">R$ {{ category.oldPrice }}</p>
+          <p class="price price-new">R$ {{ category.newPrice }}</p>
```

Comment:
If there isn't a deal here, we will not have a newPrice, so maybe put a v-if here?